src/nbl/video/ILogicalDevice.cpp

```diff
@@ -309,312 +309,102 @@ void ILogicalDevice::addCommonShaderDefines(const bool runningInRenderdoc)
-        if (features.indexTypeUint8) addShaderDefineToPool(pool, "NBL_INDEX_TYPE_UINT8");
-        if (features.shaderBufferFloat32Atomics) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT32_ATOMICS");
-        if (features.shaderBufferFloat32AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT32_ATOMIC_ADD");
-        if (features.shaderBufferFloat64Atomics) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT64_ATOMICS");
-        if (features.shaderBufferFloat64AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT64_ATOMIC_ADD");
-        if (features.shaderSharedFloat32Atomics) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT32_ATOMICS");
-        if (features.shaderSharedFloat32AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT32_ATOMIC_ADD");
-        if (features.shaderSharedFloat64Atomics) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT64_ATOMICS");
-        if (features.shaderSharedFloat64AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT64_ATOMIC_ADD");
-        if (features.shaderImageFloat32Atomics) addShaderDefineToPool(pool, "NBL_SHADER_IMAGE_FLOAT32_ATOMICS");
-        if (features.shaderImageFloat32AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_IMAGE_FLOAT32_ATOMIC_ADD");
-        if (features.sparseImageFloat32Atomics) addShaderDefineToPool(pool, "NBL_SPARSE_IMAGE_FLOAT32_ATOMICS");
-        if (features.sparseImageFloat32AtomicAdd) addShaderDefineToPool(pool, "NBL_SPARSE_IMAGE_FLOAT32_ATOMIC_ADD");
-        if (features.shaderBufferFloat16Atomics) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT16_ATOMICS");
-        if (features.shaderBufferFloat16AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT16_ATOMIC_ADD");
-        if (features.shaderBufferFloat16AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT16_ATOMIC_MIN_MAX");
-        if (features.shaderBufferFloat32AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT32_ATOMIC_MIN_MAX");
-        if (features.shaderBufferFloat64AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_BUFFER_FLOAT64_ATOMIC_MIN_MAX");
-        if (features.shaderSharedFloat16Atomics) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT16_ATOMICS");
-        if (features.shaderSharedFloat16AtomicAdd) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT16_ATOMIC_ADD");
-        if (features.shaderSharedFloat16AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT16_ATOMIC_MIN_MAX");
-        if (features.shaderSharedFloat32AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT32_ATOMIC_MIN_MAX");
-        if (features.shaderSharedFloat64AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_SHARED_FLOAT64_ATOMIC_MIN_MAX");
-        if (features.shaderImageFloat32AtomicMinMax) addShaderDefineToPool(pool, "NBL_SHADER_IMAGE_FLOAT32_ATOMIC_MIN_MAX");
-        if (features.sparseImageFloat32AtomicMinMax) addShaderDefineToPool(pool, "NBL_SPARSE_IMAGE_FLOAT32_ATOMIC_MIN_MAX");
```

Comment:
this is broken similar to your old `ILogicalDevice` PR, filter out any changes to this file (let us fix it in #584)